x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/execution/PlanExecutor.java

```diff
@@ -29,16 +29,18 @@ public class PlanExecutor {
 
     private final IndexResolver indexResolver;
     private final EnrichPolicyResolver enrichPolicyResolver;
+    // private final CircuitBreaker breaker;
     private final PreAnalyzer preAnalyzer;
     private final FunctionRegistry functionRegistry;
     private final LogicalPlanOptimizer logicalPlanOptimizer;
     private final Mapper mapper;
     private final Metrics metrics;
     private final Verifier verifier;
 
-    public PlanExecutor(IndexResolver indexResolver, EnrichPolicyResolver enrichPolicyResolver) {
+    public PlanExecutor(IndexResolver indexResolver, EnrichPolicyResolver enrichPolicyResolver) { // }, CircuitBreaker breaker) {
         this.indexResolver = indexResolver;
         this.enrichPolicyResolver = enrichPolicyResolver;
+        // this.breaker = breaker;
```

Comment:
Leftover?

Reply:
should not have been committed. Removed.